go/common/crypto/address/address.go

```diff
@@ -0,0 +1,106 @@
+// Package address implements network's generic address with versioning and
+// context separation. It can be used to implement specific network addresses,
```

Comment:
Maybe don't use the term "network address" as that sounds like a hostname or an IP address.

Reply:
Good point, thanks. I've reworded it.